ui-service/src/main/frontend/hotel-booking-ui/src/components/navigation_bar/dropdowns/NavBarSignInDropdown.tsx

```diff
@@ -0,0 +1,52 @@
+import DropdownButton from "./DropdownButton";
+
+export default function NavBarSignInDropdown() {
+  return (
+    <div className="dropdown" style={{ width: "358px" }}>
```

Comment:
This entire dropdown is set to be pointer: cursor.

Reply:
updated